Source/WebCore/html/HTMLFormControlElement.cpp

```diff
@@ -348,6 +348,8 @@ static const AtomString& hideAtom()
 HTMLElement* HTMLFormControlElement::popoverTargetElement() const
 {
     auto canInvokePopovers = [](const HTMLFormControlElement& element) -> bool {
+        if (!element.document().settings().popoverAttributeEnabled())
```

Comment:
@nt1m Isn't there also a quirk for this?

Reply:
Ah right, we should also check for `element.document().quirks().shouldDisablePopoverAttributeQuirk()`

Reply:
```suggestion
        if (!element.document().settings().popoverAttributeEnabled() || element.document().quirks().shouldDisablePopoverAttributeQuirk())
```

Reply:
Maybe we need to put this in a common method on `Document`.

Reply:
Maybe, I'm hoping to remove the quirk soon-ish though when the site gets fixed.